gwpy/plot/axes.py

```diff
@@ -143,8 +143,7 @@ def draw(self, *args, **kwargs):
                 unit = trans.get_unit_name()
                 iso = Time(epoch, format='gps', scale='utc').iso
                 utc = iso.rstrip('0').rstrip('.')
-                ax.set_label_text('Time [{0!s}] from {1!s} UTC ({2!r})'.format(
-                    unit, utc, epoch))
+                ax.set_label_text(f"Time [{unit}] from {utc} UTC ({epoch!r})")
```

Comment:
Checking here: the formatting options `s` are dropped here but not `r`. Meanwhile, on L509, the `r` format option is dropped. Can you confirm this is all fine?

Reply:
the 's' format is the default for f-strings, so it is unnecessary, while 'r' is required. On L509 the 'r' format was used to enclose the `anchor` value in quotes for the error message, but that behaviour only holds for `str` objects, it's more accurate to just use enclosing quotes manually.

Reply:
(sorry for the long-delayed reply) Sounds like you've got this understood and all is good. Resolving.